aspnetcore/grpc/comparison.md

```diff
@@ -37,15 +37,17 @@ gRPC is designed for HTTP/2, a major revision of HTTP that provides significant
 * Binary framing and compression. HTTP/2 protocol is compact and efficient both in sending and receiving.
 * Multiplexing of multiple HTTP/2 calls over a single TCP connection. Multiplexing eliminates [head-of-line blocking](https://en.wikipedia.org/wiki/Head-of-line_blocking).
 
-### Code Generation
+### Code generation
 
 All gRPC frameworks provide first-class support for code generation. A core file to gRPC development is the [`*.proto` file](https://developers.google.com/protocol-buffers/docs/proto3), which defines the contact of gRPC services and messages. From this file gRPC frameworks will code generate a service base class, messages, and a complete client.
 
 By sharing the `*.proto` file between the server and client, messages and client code can be generated from end to end. Code generation of the client eliminates duplication of messages on the client and server, and creates a strongly-typed client for you. Not having to write a client saves significant development time in applications with many services.
 
-### Strict Specification
+### Strict specification
 
-gRPC saves developer time through its simplicity. The [gRPC specification](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) is prescriptive about what a gRPC service method looks like. There is no a formal agreement of what an HTTP API with JSON should look like. The lack of an agreement creates debate over the format of URLs, HTTP verbs, and response codes. gRPC eliminates debate with a specification that states what a gRPC method must look like.
+A formal specification for HTTP API with JSON doesn't exist. Developers debate the best format of URLs, HTTP verbs, and response codes.
+
+The [gRPC specification](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) is prescriptive about the format a gRPC service must follow. gRPC eliminates debate and saves developer time through its simplicity.
```

Comment:
Be careful with "simple" and variants (e.g., "simplicity").

These words are generally **_BANNED_** on :skull: **_pain of death_** :skull: on the repo. :smile: lol ... seriously tho ... for good reason in technical writing. Such remarks can be fleshed out. For example ...

```suggestion
The [gRPC specification](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) is prescriptive about the format a gRPC service must follow. gRPC eliminates debate and saves developer time because gPRC implementations are consistent across platforms and implementations.
```

... a bit more specific.